File: ExcelDivider/src/test/java/ru/tsystems/divider/context/InMemoryComment.java
package ru.tsystems.divider.context;

import ru.tsystems.divider.dao.api.CommentDao;
import ru.tsystems.divider.entity.Comment;

public class InMemoryComment extends InMemoryDaoGeneral<Comment> implements CommentDao, ContextSimmulator<Comment>{

    private static InMemoryComment dao;

    public static InMemoryComment getInMemoryDao(){
        if(dao == null){
            dao = new InMemoryComment();
        }
        return dao;
    }

    private InMemoryComment() {
        super();
        reset();
    }
}

File: ExcelDivider/src/test/kotlin/ru/tsystems/divider/context/InMemoryComment.kt
package ru.tsystems.divider.context

import ru.tsystems.divider.dao.api.CommentDao
import ru.tsystems.divider.entity.Comment

object InMemoryComment : InMemoryDaoGeneral<Comment>(), CommentDao, ContextSimmulator<Comment> {

    init {
        reset()
    }

}